instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -0,0 +1,647 @@
+        self._system_swap_utilization_labels = self._labels.copy()
+        # self._system_swap_page_faults = self._labels.copy()
+        # self._system_swap_page_operations = self._labels.copy()
+
+        self._system_disk_io_labels = self._labels.copy()
+        self._system_disk_operations_labels = self._labels.copy()
+        self._system_disk_time_labels = self._labels.copy()
+        self._system_disk_merged_labels = self._labels.copy()
+
+        # self._system_filesystem_usage_labels = self._labels.copy()
+        # self._system_filesystem_utilization_labels = self._labels.copy()
+
+        self._system_network_dropped_packets_labels = self._labels.copy()
+        self._system_network_packets_labels = self._labels.copy()
+        self._system_network_errors_labels = self._labels.copy()
+        self._system_network_io_labels = self._labels.copy()
+        self._system_network_connections_labels = self._labels.copy()
+
+        self._runtime_memory_labels = self._labels.copy()
+        self._runtime_cpu_time_labels = self._labels.copy()
+        self._runtime_gc_count_labels = self._labels.copy()
+
+    def instrumentation_dependencies(self) -> Collection[str]:
+        return _instruments
+
+    def _instrument(self, **kwargs):
+        meter_provider = kwargs.get("meter_provider")
+        self._meter = get_meter(
+            __name__,
+            __version__,
+            meter_provider,
+        )
+
+        self._meter.create_observable_counter(
+            callback=self._get_system_cpu_time,
+            name="system.cpu.time",
+            description="System CPU time",
+            unit="seconds",
+        )
+
+        self._meter.create_observable_gauge(
+            callback=self._get_system_cpu_utilization,
+            name="system.cpu.utilization",
+            description="System CPU utilization",
+            unit="1",
+        )
+
+        self._meter.create_observable_gauge(
+            callback=self._get_system_memory_usage,
+            name="system.memory.usage",
+            description="System memory usage",
+            unit="bytes",
+        )
+
+        self._meter.create_observable_gauge(
+            callback=self._get_system_memory_utilization,
+            name="system.memory.utilization",
+            description="System memory utilization",
+            unit="1",
+        )
+
+        self._meter.create_observable_gauge(
+            callback=self._get_system_swap_usage,
+            name="system.swap.usage",
+            description="System swap usage",
+            unit="pages",
+        )
+
+        self._meter.create_observable_gauge(
+            callback=self._get_system_swap_utilization,
+            name="system.swap.utilization",
+            description="System swap utilization",
+            unit="1",
+        )
+
+        # # self._meter.create_observable_counter(
+        # #     callback=self._get_system_swap_page_faults,
+        # #     name="system.swap.page_faults",
```

Comment:
Are these TODOs?

Reply:
Grouped TODOs in the instrument method. I don't know if we need issues for each one or if we wait until someone asks for the feature?